### ISSUE TITLE
"coloring_overflow_scrollbars" example does not meet the WCAG color contrast requirement mentioned earlier in this article

### ISSUE THREAD

**Issue by jsnkuhn** [mdn/content#27140]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/CSS/scrollbar-color

### What specific section or headline is this issue about?

https://developer.mozilla.org/en-US/docs/Web/CSS/scrollbar-color#coloring_overflow_scrollbars

### What information was incorrect, unhelpful, or incomplete?

The "coloring_overflow_scrollbars" example does not meet the WCAG color contrast requirement mentioned earlier in this article: 

```css
scrollbar-color: rebeccapurple green;
```

https://developer.mozilla.org/en-US/docs/Web/CSS/scrollbar-color#accessibility_concerns

https://www.siegemedia.com/contrast-ratio#green-on-rebeccapurple

### What did you expect to see?

I'd expect that any examples on the page take into account the accessibility concerns mentioned in the article itself.

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/css/scrollbar-color`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/CSS/scrollbar-color
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/css/scrollbar-color/index.md
* Last commit: https://github.com/mdn/content/commit/622c3c65bbd007ecd59bffb5dd36e9c4c56a8f4d
* Document last modified: 2023-04-26T21:10:37.000Z

</details>
